### ISSUE TITLE
Add SKY130 example configurations?

### ISSUE THREAD

**Issue by mithro** [bespoke-silicon-group/bsg_fakeram#11]
I saw you merged a pull request ( https://github.com/bespoke-silicon-group/bsg_fakeram/pull/9 ) which mentions SKY130 but you don't seem to have any example configurations?

**Comment by mithro** [comment]
It would also be good for there to be some documentation about how the bsg_fakeram generated output for SKY130 compares to the real macros that OpenRAM provides.

**Comment by taylor-bsg** [comment]
@lakshmi-sathi Yeah I agree, I think it would be appropriate to have some diagrams that show where the pins are placed, where the metal rails are, what metal layers are blocked, etc.

@stdavids can you review Lakshmi's pull request when she does it?

**Comment by lakshmi-sathi** [comment]
I created a pull request - #12
Would you suggest any changes?

**Comment by stdavids** [comment]
Thanks for the contribution @lakshmi-sathi!

It seems like the OpenRAM abstract physical views are quite different from what fakeram naturally generates. It's outside the scope of this specific issue, but I wonder if fakeram might benefit from correlating the abstract views a bit closer with the real OpenRAM abstract views. More generally, I'm curious where fakeram's future with sky130 is (e.g. are we just a temporary placeholder while OpenRAM is in development? are there going to be continued incentives to use fakeram?)

**Comment by mithro** [comment]
In my opinion, `bsg_fakeram` can be used for research when you don't have access to memory macros or a compiler on a process technology. However, it is **only** useful if you can demonstrate that the tool is generating _reasonable_ proxies for the real macros.

The **best** way to demonstrate this is by having publicly accessible examples of `bsg_fakeram` compared to real publically accessible macros (everything else is really just hearsay and rumors). This allows people to replicate your comparisons and understand the limitations and potentially also improve the tool to be more accurate.

**Thus** it is super important to generate proxies for PDKs where you **do** have access to the compiler so you can openly document how your results compare to the real results.

Does that help?
